start.sh

```diff
@@ -5,7 +5,8 @@ cd $GITHUB_WORKSPACE
 
 if [ ! -d "node_modules" ]; then
     echo "Repo dependencies not installed. Installing..."
-    yarn install
+    [ -f yarn.lock ] && yarn install
```

Comment:
I'd rather have a if-exist check here. Right now, if both `.lock` files would exist, we'd waste resources.